src/backend/marsha/core/storage.py

```diff
@@ -1,15 +1,30 @@
 """Customizing Django storage backends to enable blue/green deployments."""
+from collections import OrderedDict
+import re
+
 from django.conf import settings
 from django.contrib.staticfiles.storage import ManifestFilesMixin, StaticFilesStorage
 
 from storages.backends.s3boto3 import S3Boto3Storage
 
 
+STATIC_NOHASH_REGEX = re.compile(getattr(settings, "STATIC_NOHASH_PATTERN", "^$"))
```

Comment:
I thought of another name:
`STATIC_POSTPROCESS_IGNORE_REGEX` or `STATIC_POSTPROCESS_EXCLUDE_REGEX`